.github/husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Be sure to run `chmod +x` on this and the other file through WSL so they are executable for unix contributors.